test/main.js

```diff
@@ -31,6 +32,48 @@ var options = {
   templateDonut: {
     'twig': './templates/donut.twig'
   },
+  convertCategoryTitles: true,
+  convertCategoryTitlesData: {
+    "categories": {
+      "base": "00-atoms",
```

Comment:
Also, those "00", ... numerations are just for folder and menu order in PL, we can omit them and have everything sorted alphabetically, it's not critical.